steps/api/epf/epf-api.ts

```diff
@@ -0,0 +1,20 @@
+import { type APIRequestContext, expect, request } from '@playwright/test'
+import { getToken } from '../auth/get-token.js'
+
+async function getContext(): Promise<APIRequestContext> {
+    const token = await getToken()
```

Comment:
I think this uses my personal dev client, I'll ask the auth team to add the EPF_CONTEXT role now

Reply:
I asked Matt to add them everywhere we need them so he might have done it